tutorials/README.md

```diff
@@ -0,0 +1,15 @@
+# Tutorial
+
+This folder contains simple tutorials showcasing basic features of the `3lc` Python package.
+
+## Overview
+
++ [Register Image Classification Dataset](write-image-classification-table.ipynb) Write a table consisting of images and labels.
++ [Add Image Metrics](add-image-metrics.ipynb) Extend an existing table with image metrics.
++ [Add Embeddings](add-embeddings.ipynb) Extend an existing table with embeddings from a pre-trained model.
++ [Register Augmented Samples](write-augmented-samples.ipynb) Register images as metrics on-the-fly, using image augmentation as an example.
++ [Collect Metrics from a Pre-trained Model](collect_metrics_only) Collect classification metrics from a pre-trained model.
++ [Write Video Thumbnails](write-video-thumbnails.ipynb) Write a table consisting of video thumbnails and paths to videos.
++ [Tutorial 2](write-bb-table.ipynb) Write a table consisting of images and random bounding boxes.
```

Comment:
"Tutorial 2" is not a very descriptive name. Can this be changed to reflect what the tutorial actually does?